201-vmss-automation-dsc/nested/provisionConfiguration.json

```diff
@@ -0,0 +1,126 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "_artifactsLocation": {
+      "type": "string"
+    },
+    "_artifactsLocationSasToken": {
+      "type": "string"
+    },
+    "automationAccountName": {
+      "type": "string"
+    },
+    "location": {
+      "type": "string"
+    },
+    "compileName": {
+      "type": "string",
+      "defaultValue": "[guid(resourceGroup().id, deployment().name)]"
+    }
+  },
+  "variables": {
+    "dscConfigurations": {
+      "WindowsIISServerConfig": {
+        "name": "WindowsIISServerConfig",
+        "description": "minimum viable configuration for a web server role",
+        "script": "dscConfigurations/WindowsIISServerConfig.ps1"
+      }
+    },
+    "dscResources": {
+      "xWebAdministration": {
+        "name": "xWebAdministration",
+        "url": "https://psg-prod-eastus.azureedge.net/packages/xwebadministration.2.4.0.nupkg"
+      }
+    },
+    "runbook": {
+      "Name": "TombstoneDSCNodes",
+      "Uri": "https://raw.githubusercontent.com/mgreenegit/tombstonedscnodes/master/TombstoneDSCNodes.ps1",
```

Comment:
add this to the sample and pass in the relative path during deployment (use the _artifactsLocation)

Reply:
Is it acceptable to not?  I would rather not get on "a treadmill" of keeping it up to date in both places.

Reply:
@bmoore-msft

Reply:
It needs to be in this repo, otherwise we end up with samples that have orphaned artifacts (and we have no choice but to delete them).

Reply:
got it.  fixed in latest commit.